tests/create_tests_of_examples.py

```diff
@@ -10,22 +10,37 @@
 
 download_assets()
 
+EXAMPLES_TO_SKIP = [
+    "action_primitives.rs_int_example",
+    "action_primitives.solve_simple_task",
+    "action_primitives.wip_solve_behavior_task",
+    "learning.navigation_policy_demo",
+    "teleoperation.robot_teleoperate_demo",
+    # TODO: Temporarily skip the following examples
+    # "object_states.attachment_demo",  # seg fualt??
+    # "environments.behavior_env_demo",  # This only works with pre-sampled cached BEHAVIOR activity scene
+]
+
 
 def main():
     examples_list = []
-    for package in pkgutil.walk_packages(examples.__path__, examples.__name__ + "."):
+    prefix = examples.__name__ + "."
+    for package in pkgutil.walk_packages(examples.__path__, prefix):
         if (
             not package.ispkg
-            and package.name[17:] != "example_selector"
-            and "web_ui" not in package.name[17:]  # The WebUI examples require additional server setup
-            and "vr_" not in package.name[17:]  # The VR examples require additional dependencies
-            and "ray_" not in package.name[17:]  # The Ray/RLLib example does not run in a subprocess
-        ):  # Consider removing the last condition if we have runnable VR tests
-            examples_list += [package.name[17:]]
+            and package.name[len(prefix) :] != "example_selector"
+            and "web_ui" not in package.name[len(prefix) :]
+            and "vr_" not in package.name[len(prefix) :]
+            and "ray_" not in package.name[len(prefix) :]
```

Comment:
these might need to change?

Reply:
Removing these filters; these are included now with `EXAMPLES_TO_SKIP` (more verbose for which examples to exclude)